.github/workflows/python-app.yml

```diff
@@ -45,6 +45,10 @@ jobs:
           pre-commit run --hook-stage commit-msg --commit-msg-filename commit-msg.txt --files commit-msg.txt || exit 1
```

Comment:
I would pass the hook id so that only that hook is used and the output of the command is cleaner:

`pre-commit run conventional-pre-commit --hook-stage commit-msg --commit-msg-filename commit-msg.txt`

I don't think `--files commit-msg.txt` is used and it can be removed.